docroot/modules/custom/cgov_glossary/config/install/cgov_glossary.settings.yml

```diff
@@ -0,0 +1 @@
+host_name: 'https://webapis.cancer.gov/glossary/v1'
```

Comment:
Where is this being used? I do not see it referenced anywhere. I also have concerns if this is to be used for by PHP code, but the front-end will use a TOTALLY different parameter. Please remove this for now. 

And yes, I see it looks like CTS does have different settings for front-end and back-end and I do not like that... Let's not repeat that oversight.

Reply:
I thought perhaps we'd need this value for setting up the calls to the API. That's covered by the app module config, so I've deleted this file and the schema file.